### ISSUE TITLE
[Bug]: System prompt has no effect using Ollama

### ISSUE THREAD

**Issue by PLEXATIC** [run-llama/llama_index#13983]
### Bug Description

If a system prompt is used when creating an instance of the Ollama_llm class, one can pass the parameter system_prompt. However, this parameter is seemingly not used in generation down the line and has absolutely no effect on the generated outputs.

### Version

0.10.34

### Steps to Reproduce

- Host your own Ollama LLM Service. (I used phi3-mini)
- Instantiate Ollama Class:
`ollama_llm = Ollama(model="phi3", temperature=0.0, system_prompt="You are german")`
- Make a prediction:
`pred = ollama_llm.complete("Name an amazing beverage!")`

If we print pred, we will see the model reacts in english. We can change the system prompt however we like, the result will allways stay the same- The sysprompt has absolutely no effect on the response that is generated.
Given that I instantiated my LLM-Object WITH a system-prompt, I would definitely expect the LLM to actually make use of this property, otherwise why would I even be given the option to pass it? Perhaps I used a wrong method or missed a parameter to actually make use of the sysprompt, but I could not find anything in the docs.

### Relevant Logs/Tracbacks

```shell
If I use 'ollama run' to run phi3 in console, I can set a system prompt there, and it will influence the generated text a lot and works as expected, so the error must be somewhere in Llama-Index.
I checked briefly, there does seem to be a mechanism to prepend the sysprompt. However, this is seemingly not being used for the Ollama LLMs...
```

**Comment by logan-markewich** [comment]
This is not a bug sadly -- the system prompt param is only used for llm.predict/stream. Tbh I've been meaning to remove this, or at least make this more clear 

If you want a system prompt, use llm.chat

```
from llama_index.core.llms import ChatMessage

response = llm.chat([ChatMessage(role="system", content="system prompt"), ChatMessage(role="user", content="user prompt")])
```

**Comment by PLEXATIC** [comment]
Thats the solution I also came up with...
I don't understand, why would it simply ignore the system prompt value in llm.complete()? I mean I am totally fine with it not applying a system prompt if I dont pass one, which is an option. But the way it currently is, someone reading the code will very clearly think "oh nice, I can just add a system prompt here and I will be able to use it in chat/complete etc.".

Also I doubt that this is entirely true. I was able to find some _extend_prompt functions, so it is definitely intended for a chat or something to have a system prompt, but something seems to be wrong.

If the system prompt property is not used in all text-generation functions, it should be written clearly in the docs, or it could also be possible to pass system prompt only to functions that actually need it. Otherwise, people like me are left with a very valid expectation that if I create an object and pass a value, I expect it to be used and I dont think that is a crazy thing to demand...
